FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -7,6 +7,7 @@ import Foundation
 public final class RemoteFeedLoader: FeedLoader {
 	private let url: URL
 	private let client: HTTPClient
+	private let OK_200 = 200
```

Comment:
This property could be `static`, so we can access it without an `instance`. Also, there's no reason for each instance of `RemoteFeedLoader` to have it's own value for `OK_200`.
```suggestion
	private static let OK_200 = 200
```